stingray/covariancespectrum.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import division
+
+import numpy as np
+
+from stingray import Lightcurve
+import stingray.utils as utils
+
+
+class Covariancespectrum(object):
+
+    def __init__(self, event_list, bin_width, band_interest=None,
+                 ref_band_interest=None):
+        """
+        Parameters
+        ----------
+        event_list : numpy 2D array
+            A numpy 2D array with first column as time of arrival and second
+            column as photon energies associated.
+
+        bin_width : float
+            The width of the energy bin when the spectrum is divided into N
+            divisions.
+
+        band_interest : iterable of tuples, default All
+            An iterable of tuples with minimum and maximum values of the range
+            in the band of interest.
+
```

Comment:
Maybe I named it wrong but the `bin_width` is being used as `dt` in constructing the lightcurves below.